src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -151,19 +151,24 @@ export default class SentryApplicationDashboard extends AsyncView<Props, State>
 
           <PanelBody>
             {requests.length > 0 ? (
-              requests.map((request, idx) => (
-                <PanelItem key={idx}>
-                  <TableLayout>
-                    <DateTime date={request.date} />
-                    <div>{request.responseCode}</div>
-                    {app.status !== 'internal' && request.organization && (
-                      <div>{request.organization.name}</div>
-                    )}
-                    <div>{request.eventType}</div>
-                    <OverflowBox>{request.webhookUrl}</OverflowBox>
-                  </TableLayout>
-                </PanelItem>
-              ))
+              requests.map((request, idx) => {
+                const LinkOrDiv = request.errorUrl ? Link : 'div';
+                const linkTo = request.errorUrl;
+
+                return (
+                  <PanelItem key={idx}>
+                    <TableLayout>
+                      <DateTime date={request.date} />
+                      <LinkOrDiv to={linkTo}>{request.responseCode}</LinkOrDiv>
```

Comment:
@nolachen this is a bit wonky IMO since `to` doesn't mean anything in the context of a div.

Reply:
@scefali actually i have another PR out that has some UI changes for the request log so i'll just delete all the UI stuff from this one for now